### PR TITLE
ALL Features implemented. Merged everyone's code. Final Commit

### PR DESCRIPTION
**Pull request by pollchowdhury**
Merging Final commit. Merging mine, Jaideep's, Mahnoor's, Andrea, and Reem's project contribution.

### CODE REVIEW

**Comment by pollchowdhury** [comment]
Merged Final commit.

**Comment by jchunduri35** [comment]
Merged all features.

File: src/Components/Pages/Table.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  console.log(results);
+  useEffect(() => {
+    for (var i = 0; i < results.length; i++) {
+      SetPartNum(results[i].PartNum);
+      SetProductLink(results[i].ProductLink);
+      SetProductVideo(results[i].ProductVideo);
+    }
+  }, []);
+  return (
+    <>
```
Comment (reemrizk) [inline src/Components/Pages/Table.js:25]: Add a bit of documentation to organize processes

**Review by reemrizk — COMMENTED** [review]
Code review

File: src/Components/Pages/AtHome.js [hunk 2]
```diff
@@ -1,12 +1,95 @@
...
+              sx={{ textDecoration: "none" }}
+              variant="contained"
+              as={Link}
+              to={"/Lights"}
+            >
+              Lights Replacement
+            </Button>
 
-export default AtHome;
+            <Button
```
Comment (reemrizk) [inline src/Components/Pages/AtHome.js]: Could move buttons into separate component

File: src/Components/Pages/Tires/Fluid.js [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+              </MenuItem>
+              <MenuItem value="PowerSteeringFluid">
+                Power Steering Fluid
+              </MenuItem>
+              <MenuItem value="WindshieldWiperFluid">
+                Windshield Wiper Fluid
+              </MenuItem>
+            </Select>
+          </Stack>
+          {(FluidTypes === "PowerSteeringFluid" ||
```
Comment (reemrizk) [inline src/Components/Pages/Tires/Fluid.js]: Adding a bit of documentation can help to clearly read content

**Review by reemrizk — COMMENTED** [review]
code review

**Pull request merged**